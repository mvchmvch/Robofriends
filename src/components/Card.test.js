// test file
import React from 'react'
import Card from './Card'
import { shallow, mount, render } from 'enzyme';

it("test Card", () => {
	expect(shallow(<Card />)).toMatchSnapshot();
 
})
