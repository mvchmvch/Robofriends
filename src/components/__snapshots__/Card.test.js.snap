// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Card 1`] = `ShallowWrapper {}`;
